packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+rpm2deb() {
+	set -x
+	chown build:build /home/build
+	pushd /debbuild/PKG 1>/dev/null
+	for RPM in /rpmbuild/SRC/calico*.rpm; do
+		alien --bump=0 "$RPM"
+	done
+	cp -p /rpmbuild/SRC/*.deb /debbuild/PKG
+	chown "${TARGET_UID}:${TARGET_GID}" \
+		/debbuild/DEBS/*
+}
+
+builddeb() {
+	#need to be done with user "build"
+	set -x
+	chown build:build /home/build
+        pushd /debbuild/PKG 1>/dev/null
+	for DEBSRC in /debbuild/PKG; do
+		pushd /debbuild/PKG/metalk8s-sosreport/debian
+		debuild -b -uc -us
+	done
+	cp -p /debbuild/SRC/* /debbuild/PKG
```

Comment:
Not really an error, but is there any good reason for using `-p` ? (just for my personal understanding)

Reply:
It was to "preserve" rights on its but no, it was not very needed because of the `chown` applied after.